titus_isolate/monitor/workload_monitor_manager.py

```diff
@@ -23,16 +23,22 @@ def __init__(self, resource_usage_provider: ResourceUsageProvider):
         self.__registry = None
         self.__lock = Lock()
         self.__usage_lock = Lock()
+        self.__get_resource_usage_failure_count = 0
 
     def __get_usage_dict(self, workload_ids: List[str]) -> dict:
         log.info("Getting resource usage from resource usage provider: %s", self.__resource_usage_provider.get_name())
         usages_dict = resource_usages_to_dict(self.__resource_usage_provider.get_resource_usages(workload_ids))
         return usages_dict
 
     def get_resource_usage(self, workload_ids: List[str]) -> GlobalResourceUsage:
-        global_usage = GlobalResourceUsage(self.__get_usage_dict(workload_ids))
-        log.debug("Got resource usage: %s", json.dumps(global_usage.serialize(), sort_keys=True, separators=(',', ':')))
-        return global_usage
+        try:
+            global_usage = GlobalResourceUsage(self.__get_usage_dict(workload_ids))
+            log.debug("Got resource usage: %s", json.dumps(global_usage.serialize(), sort_keys=True, separators=(',', ':')))
+            return global_usage
+        except:
```

Comment:
`except Exception:` at least so that it behaves correctly on things like kill signals

https://stackoverflow.com/questions/54948548/what-is-wrong-with-using-a-bare-except